### DISCUSSION TITLE
Adaptable parameters for handling diverse query strings

### DISCUSSION THREAD

**Discussion by graceVanderwall** [ilhamfadllah/rss-news-to-json-node-sdk#2] [Ideas]
Hello, thank you for developing this SDK. As a novice researcher diving into RSS, I'm considering the need for accommodating flexible query strings, especially in the context of various news providers. This feature would greatly assist SDK users in aligning with third-party RSS specifications.

**Comment by ilhamfadllah** [comment]
Hi,

Thanks for the idea. It will be more versatile, as every provider has different specs. I will try to roll it out in the next iteration. I believe Google News RSS is a good example to test flexible query strings.
